client/state/ui/olark/selectors.js

```diff
@@ -51,3 +51,21 @@ export function isChatAvailable( state, context ) {
 export function isOperatorsAvailable( state ) {
 	return state.ui.olark.operatorStatus === OPERATOR_STATUS_AVAILABLE;
 }
+
+/**
+ * Returns if precancellation chat is available.
+ * @param   {Object}  state  Global state tree
+ * @returns {Boolean}        true, when precancellation is available
+ */
+export function isCancellationChatAvailable( state ) {
```

Comment:
The same idea from the above comment, let's rename "cancellation" as "precancellation" :)

Reply:
Sounds good! Will update.